subprojects/packagefiles/openssl/generator.sh

```diff
@@ -31,11 +31,16 @@ rm -rf config/archs
 LANG=C make -C config
 
 # Copy generated files back into correct place
-find config/archs -name 'meson.build' | xargs -I % sh -c 'mkdir -p ../../../generated-$(dirname %); cp % ../../../generated-%'
-find config/archs -name '*.asm' | xargs -I % sh -c 'mkdir -p ../../../generated-$(dirname %); cp % ../../../generated-%'
-find config/archs -name '*.c' | xargs -I % sh -c 'mkdir -p ../../../generated-$(dirname %); cp % ../../../generated-%'
-find config/archs -name '*.h' | xargs -I % sh -c 'mkdir -p ../../../generated-$(dirname %); cp % ../../../generated-%'
-find config/archs -iname '*.s' | xargs -I % sh -c 'mkdir -p ../../../generated-$(dirname %); cp % ../../../generated-%'
+cmd () {
+  mkdir -p ../../../generated-$(dirname "$1")
+  cp "$1" ../../../generated-"$1"
+}
+export -f cmd
+find config/archs -name 'meson.build' -exec sh -c "cmd {}" \;
```

Comment:
I mentioned in the other ticket how this could be done without `export -f`, but if you're going to use `export -f` then you also need to explicitly run `-exec bash -c` because "sh" might be a shell that doesn't implement this, such as dash or busybox.

Reply:
Fixed. Removed export -f for more portable sh friendly approach.